compliance/compliance.go

```diff
@@ -91,17 +94,34 @@ func (c *Compliance) Start() {
 		c.manageStream(ctx, cli, &stoppedSig, toSensorC)
 	}()
 
-	if c.nodeScanner.IsActive() || env.NodeIndexEnabled.BooleanSetting() {
-		nodeInventoriesC := c.manageNodeScanLoop(ctx)
-		// sending nodeInventories into output toSensorC
-		for n := range nodeInventoriesC {
-			toSensorC <- n
+	wg := &sync.WaitGroup{}
```

Comment:
nit: I think `var wg sync.WaitGroup` is sufficient, too

Reply:
There were only 2 occurrences of `wg := &sync.WaitGroup{}` in the entire codebase, so I replaced them both with `var wg sync.WaitGroup`.